CMakeLists.txt

```diff
@@ -24,7 +24,7 @@ endif()
 
 set(OTHER_INCLUDES ${EIGEN3_INCLUDE_DIRS} ${Pangolin_INCLUDE_DIRS})
 
-include_directories(include ${CMAKE_CURRENT_BINARY_DIR}/include)
+include_directories(include ${CATKIN_DEVEL_PREFIX}/include)
```

Comment:
I think what you want is a separate `include_directories(${catkin_INCLUDE_DIRS})` :wink:

Reply:
From @meyerj comment I think it is correct.

> Do not forget to include_directories(${CATKIN_DEVEL_PREFIX}/include) and catkin_package(INCLUDE_DIRS ${CATKIN_DEVEL_PREFIX}/include). Both are not added automatically by catkin macros.